runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -5950,6 +5951,263 @@ static TR::Register *inlineCRC32CUpdateBytes(TR::Node *node, TR::CodeGenerator *
    return inputReg;
    }
 
+/**
+ * @brief Generates vectorized instruction sequence for computing hashcode of a String
+ *
+ * @param[in] node: node
+ * @param[in] isCompressed: true if compressed string
+ * @param[in] cg: CodeGenerator
+ *
+ * @returns register
+ */
+static TR::Register* inlineStringHashCode(TR::Node* node, bool isCompressed, TR::CodeGenerator* cg)
```

Comment:
Please don't mix the `type* variable` style and the `type *variable` style.
The variables declared later in this function use the second style.

Reply:
Fixed.